.github/workflows/setup-tofu.yml

```diff
@@ -0,0 +1,314 @@
+        tofu_wrapper: false
+
+    - name: Validate OpenTofu Version - ${{ matrix['tofu-versions'] }}
+      if: ${{ matrix['tofu-versions'] != 'latest' }}
+      run: tofu version | grep ${{ matrix['tofu-versions']}}
+
+    - name: Validate OpenTofu Version - ${{ matrix['tofu-versions'] }}
+      if: ${{ matrix['tofu-versions'] == 'latest' }}
+      run: tofu version | grep 'OpenTofu v'
+
+# TODO: uncomment when the semver feature is implemented
+#  tofu-versions-constraints:
+#    name: 'OpenTofu Versions Constraints'
+#    runs-on: ${{ matrix.os }}
+#    strategy:
+#      matrix:
+#        os: [ubuntu-latest, windows-latest, macos-latest]
+#        tofu-versions: [~0.12, 0.12.x, <0.13.0]
+#    steps:
+#    - name: Checkout
+#      uses: actions/checkout@3df4ab11eba7bda6032a0b82a6bb43b11571feac # v4.0.0
+#
+#    - name: Setup OpenTofu - ${{ matrix['tofu-versions'] }}
+#      uses: ./
+#      with:
+#        tofu_version: ${{ matrix['tofu-versions'] }}
+#
+#    - name: Validate OpenTofu Version - ${{ matrix['tofu-versions'] }}
+#      run: tofu version | grep 'OpenTofu v0\.12'
+
+# TODO: uncomment when the semver feature is implemented
+#  tofu-versions-constraints-no-wrapper:
+#    name: 'OpenTofu Versions Constraints No Wrapper'
+#    runs-on: ${{ matrix.os }}
+#    strategy:
+#      matrix:
+#        os: [ubuntu-latest, windows-latest, macos-latest]
+#        tofu-versions: [~0.12, 0.12.x, <0.13.0]
+#    steps:
+#    - name: Checkout
+#      uses: actions/checkout@3df4ab11eba7bda6032a0b82a6bb43b11571feac # v4.0.0
+#
+#    - name: Setup OpenTofu (no wrapper) - ${{ matrix['tofu-versions'] }}
+#      uses: ./
+#      with:
+#        tofu_version: ${{ matrix['tofu-versions'] }}
+#        tofu_wrapper: false
+#
+#    - name: Validate OpenTofu Version - ${{ matrix['tofu-versions'] }}
+#      run: tofu version | grep 'OpenTofu v0\.12'
+
+  tofu-credentials-cloud:
+    name: 'OpenTofu Cloud Credentials'
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+    env:
+      TF_CLOUD_API_TOKEN: 'XXXXXXXXXXXXXX.atlasv1.XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
+    steps:
+    - name: Checkout
+      uses: actions/checkout@3df4ab11eba7bda6032a0b82a6bb43b11571feac # v4.0.0
+
+    - name: Setup OpenTofu
+      uses: ./
+      with:
+        cli_config_credentials_token: ${{ env.TF_CLOUD_API_TOKEN }}
+
+    - name: Validate OpenTofu Credentials (Windows)
+      if: runner.os == 'Windows'
+      run: |
+        cat ${APPDATA}/terraform.rc | grep 'credentials "app.terraform.io"'
+        cat ${APPDATA}/terraform.rc | grep 'token = "${{ env.TF_CLOUD_API_TOKEN }}"'
+
+    - name: Validate Teraform Credentials (Linux & macOS)
+      if: runner.os != 'Windows'
+      run: |
+        cat ${HOME}/.terraformrc | grep 'credentials "app.terraform.io"'
+        cat ${HOME}/.terraformrc | grep 'token = "${{ env.TF_CLOUD_API_TOKEN }}"'
```

Comment:
Consider supporting OpenTofu's `.tofurc` or `tofu.rc` [CLI Configuration File](https://opentofu.org/docs/cli/config/config-file).

Reply:
Great catch! Thank you!

Reply:
Implemented